src/kubectl-helm-minikube/devcontainer-feature.json

```
@@ -1,6 +1,6 @@
 {
     "id": "kubectl-helm-minikube",
-    "version": "1.1.10",
+    "version": "1.1.11",
```

```suggestion
    "version": "1.2.0",
```

Let's bump minor version as we are adding new utils.